ethereum/performance-trackers/src/main/java/tech/pegasys/teku/ethereum/performance/trackers/BlockProductionPerformanceImpl.java

```diff
@@ -21,22 +21,39 @@
 
 public class BlockProductionPerformanceImpl implements BlockProductionPerformance {
 
+  public static final String PREPARATION_ON_TICK = "preparation_on_tick";
+  public static final String PREPARATION_APPLY_DEFERRED_ATTESTATIONS =
+      "preparation_apply_deferred_attestations";
+  public static final String PREPARATION_PROCESS_HEAD = "preparation_process_head";
+  public static final String BEACON_BLOCK_PREPARED = "beacon_block_prepared";
+  public static final String RETRIEVE_STATE = "retrieve_state";
+  public static final String LOCAL_GET_PAYLOAD = "local_get_payload";
+  public static final String BUILDER_GET_HEADER = "builder_get_header";
+  public static final String BUILDER_BID_VALIDATED = "builder_bid_validated";
+  public static final String BEACON_BLOCK_CREATED = "beacon_block_created";
+  public static final String STATE_TRANSITION = "state_transition";
+  public static final String STATE_HASHING = "state_hashing";
+  public static final String GET_ATTESTATIONS_FOR_BLOCK = "get_attestations_for_block";
+  public static final String BEACON_BLOCK_PREPARATION_STARTED = "beacon_block_preparation_started";
+  public static final String TOTAL_PRODUCTION_TIME_LABEL = "total_production_time";
```

Comment:
should this just be an enum?

Reply:
It could be, just following the same notation we used in `BlockImportPerformance` changing them will imply changes on some other classes that consume those though.